vm-virtio/src/net_util.rs

```diff
@@ -2,21 +2,29 @@
 //
 // SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
 
+use super::Error as DeviceError;
 use super::{DescriptorChain, DeviceEventT, Queue};
+use arc_swap::ArcSwap;
 use net_util::{MacAddr, Tap, TapError, MAC_ADDR_LEN};
 use std::cmp;
-use std::io::Write;
+use std::io::{self, Write};
 use std::mem;
 use std::net::Ipv4Addr;
+use std::os::unix::io::{AsRawFd, RawFd};
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
 use virtio_bindings::bindings::virtio_net::*;
-use vm_memory::{Bytes, GuestAddress, GuestMemoryMmap};
+use vm_memory::{Bytes, GuestAddress, GuestMemoryError, GuestMemoryMmap};
+use vmm_sys_util::eventfd::EventFd;
 
 type Result<T> = std::result::Result<T, Error>;
 
 /// The maximum buffer size when segmentation offload is enabled. This
 /// includes the 12-byte virtio net header.
 /// http://docs.oasis-open.org/virtio/virtio/v1.0/virtio-v1.0.html#x1-1740003
 const MAX_BUFFER_SIZE: usize = 65562;
+const VIRTIO_NET_CTRL_MQ: u8 = 4;
+const QUEUE_SIZE: usize = 256;
```

Comment:
Is this only used for the control queue?

Reply:
yep, they are only for control queue.